inst/stan/wwinference.stan

```diff
@@ -87,6 +89,15 @@ data {
   real<lower=0> log_phi_g_prior_sd;
   real inf_feedback_prior_logmean;
   real<lower=0> inf_feedback_prior_logsd;
+
+  real log_phi_mu;
```

Comment:
I would label these as priors e.g. `log_phi_mu_prior`. Also, we probably should rename either this parameter or our current `phi` parameter which is the dispersion in the negative binomial hospital admissions observation process.